pkg/gsclient/types.go

```diff
@@ -5,13 +5,11 @@ type ClusterEntry struct {
 	ReleaseVersion string `json:"release_version"`
 }
 
-// TODO: Use the gsctl type directly
```

Comment:
I don't think it's worth trying to use `gsctl` directly here at this point. The api isn't intended for it and we'll be moving to kubectl-gs anyway.